augur/api/config.py

```diff
@@ -37,6 +37,8 @@
 
 from augur.application.logs import ROOT_AUGUR_DIRECTORY
 
+loggy_var=ROOT_AUGUR_DIRECTORY+'/logs'
```

Comment:
@IsaacMilarky I'm assuming this is your change. I think the logs dir may need a `/` at the end. I'd at least like to check before this is merged.

Reply:
@IsaacMilarky / @ABrain7710 : I confirmed there is a trailing slash when its stored in the database. That is working, so ... probably keep it.